src/fauxtoshop.cpp

```diff
@@ -425,16 +393,20 @@ void applyEdgeDetectionFilter(const Grid<int> &original, Grid<int> &blackWhiteGr
 */
 
 //Grid<int> applyGreenScreenFilter(GBufferedImage &sticker, GWindow &gw, int& row, int& col){

```

Comment:
Don't forget to remove any unused comments or code.

Reply:
Fixed!